vm/systemSmartContracts/esdt.go

```diff
@@ -770,6 +771,16 @@ func (e *esdt) upgradeProperties(token *ESDTDataV2, args [][]byte, isCreate bool
 		}
 	}
 
+	topics := make([][]byte, 0)
+	topics = append(topics, tokenIdentifier, big.NewInt(0).Bytes())
```

Comment:
why hardcoded to 0? Is this the number of decimals? Can we have this in a local variable?

Reply:
topics[1] --- is the `nonce` in order to have similar logs  like the ones generated by the SFTs, NFTs and Meta ESDTs